source/making_payments/index.html.md.erb

```diff
@@ -77,37 +100,36 @@ datastore mapped to the payment ID just after you create a payment.
  * not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
-
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, two hours).
+Users may close their browser or lose internet connection in the middle of
```

Comment:
This either wants to be “lose _their_ internet connection” or “lose internet connectivity”.

Reply:
Indeed, connectivity